website/docs/docs/partnerships.mdx

```diff
@@ -154,15 +146,17 @@ HashiCorp Terraform has an extremely wide community of users and contributors, a
+
 ## Terraform Cloud Integrations
 
-Run tasks allow Terraform Cloud to execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can add and execute these tasks during the pre-plan (public beta) and post-plan run stages. To execute a run task, Terraform Cloud sends an API payload to the external system. This payload contains a collection of run-related information and a callback URL that the external system can use to send updates back to Terraform Cloud.
+Run tasks allow Terraform Cloud to execute tasks in external systems at certain points in the Terraform Cloud run lifecycle. Specifically, Terraform Cloud users can add and execute these tasks during the pre-plan (public beta), pre-apply (public beta) and post-plan run stages. To execute a run task, Terraform Cloud sends an API payload to the external system. This payload contains a collection of run-related information and a callback URL that the external system can use to send updates back to Terraform Cloud.
 
 The external system can then use this run information and respond back to Terraform Cloud with a passed or failed status. Terraform Cloud uses this status response to determine if a run should proceed, based on the task's enforcement settings within a workspace.
 
-Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This signifies HashiCorp has verified the integration and the partner is a member of the HashiCorp Technology Partner Program.
+Partners who successfully complete the Terraform Cloud integration checklist obtain a Terraform Cloud badge. This signifies HashiCorp has validated the integration under ideal conditions and the partner is a member of the HashiCorp Technology Partner Program.
 
-- Note: As of September 2022, pre-plan run tasks are available only as a public beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization. This feature is enabled by default for our business tier customers of Terraform Cloud. If you have a customer that is interested in run tasks and are not a current Terraform Cloud for Business customer, please ask them to [sign up for the public beta](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
+- Note: As of September 2022, pre-plan & pre-apply run tasks are available only as a public beta feature, are subject to change, and not all customers will see this functionality in their Terraform Cloud organization. This feature is enabled by default for our business tier customers of Terraform Cloud. If you have a customer that is interested in run tasks and are not a current Terraform Cloud for Business customer, please ask them to [sign up for the public beta](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
```

Comment:
```suggestion
- Note: As of September 2022, pre-plan and pre-apply run tasks are available only as a public beta feature, are subject to change, and not all customers see this functionality in their Terraform Cloud organization. This feature's enablement is the default for our Terraform Cloud Business tier customers. If you have a customer who wants to run tasks and are not a current Terraform Cloud Business tier customer, please ask them to [sign up for the public beta](https://docs.google.com/forms/d/e/1FAIpQLSf3JJIkU05bKWov2wXa9c-QV524WNaHuGIk7xjHnwl5ceGw2A/viewform).
```
Suggest avoid ampersands and using the word "and" instead. Suggest removing "will" to avoid future tense. Suggest changing "This feature is enabled by default" to "This feature's enablement is the default" to avoid a passive voice style issue. Suggest modifying the tier name to "Terraform Cloud Business tier" for uniformity. Suggest changing "is interested in" to "wants to" to avoid a passive voice style issue.